python/pyspark/pandas/frame.py

```diff
@@ -11141,9 +11141,12 @@ def all(
 
         return self._result_aggregated(column_labels, applied)
 
-    # TODO(SPARK-46166): axis and **kwargs should be implemented.
     def any(
-        self, axis: Axis = 0, bool_only: Optional[bool] = None, skipna: bool = True
+        self,
+        axis: Optional[Axis] = 0,
+        bool_only: Optional[bool] = None,
+        skipna: bool = True,
+        **kwargs: Any,
```

Comment:
it seems `kwargs` are not used/passed in the PR, and don't have a test for it

Reply:
I cannot find an example with kwargs in https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.any.html,
not sure how it works in pandas.

But in PS, I prefer not adding it if it doesn't take effect.